osler/templates/surveys/fill.html

```diff
@@ -0,0 +1,115 @@
+{% extends "core/base.html" %}
+{% load bootstrap3 %}
+
+{% block title %}
+Osler: Surveys
+{% endblock %}
+
+
+{% block content %}
+
+<style>
+/*not sure how to compile project.scss so putting this here for now*/
+.required-star {
+  color: red;
+  font-size: large;
+}
+</style>
```

Comment:
should this be in its own file?

Reply:
Honestly, I don't really know very much about css (or how to compile scss) so I think this is fine for now. If you figure out how css works you will be a hero, but leaving this here for now is fine with me.